deployment/ansible/roles/model-my-watershed.geotrellis/meta/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+dependencies:
+  - { role: "model-my-watershed.base" }
+  - { role: "azavea.git" }
+  - { role: "azavea.spark", spark_version: "1.3.1-bin-hadoop2.6" }
```

Comment:
`spark_version` here should probably be moved up into `group_vars/all`. Also, I think we should be up to `1.4.0-bin-hadoop2.6`, unless we were told to pin to 1.3.1.